newIDE/app/src/UI/List.js

```diff
@@ -265,7 +265,7 @@ export const ListItem = React.forwardRef<ListItemProps, ListItemRefType>(
             ) : null}
             {renderListItemSecondaryAction()}
           </MUIListItem>
-          {isItemOpen && (
+          <Collapse in={isItemOpen} timeout="auto" unmountOnExit>
```

Comment:
Could you verify if the unmountOnExit effectively works? I don't see it in the version of Material-UI we're using currently.

Also this List is used in other places that in the Project Manager. Can you verify what the result in other places using a `List` with nested items (typically, the action/condition selector maybe?).

Thanks!

Reply:
I can confirm that the unmountOnExit works, you can see below that in the dev tools the component is unmounted when the list is closed otherwise it would just be hidden

No unmountOnExit:

https://user-images.githubusercontent.com/73597906/144232339-35d72cc9-a5dc-4d79-98ff-efab451f8e2d.mp4

With unmountOnExit:

https://user-images.githubusercontent.com/73597906/144232374-4fa7a1fc-3c99-4405-a542-ba496fc160b0.mp4

As far as I have tested the list works fine in the actions editor too;


https://user-images.githubusercontent.com/73597906/144232453-ce201595-9e1f-4185-a107-7aac057cedd8.mp4

Reply:
Thanks for checking! Sounds good then, let's give this a try :)